package.json

```diff
@@ -29,28 +29,28 @@
     "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
   },
   "dependencies": {
-    "@babel/core": "^7.16.12",
-    "@babel/eslint-parser": "^7.16.5",
-    "eslint-config-prettier": "^8.5.0",
+    "@babel/core": "^7.21.4",
+    "@babel/eslint-parser": "^7.21.3",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-mocha": "^10.1.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-promise": "^6.0.1",
-    "eslint-plugin-rulesdir": "^0.2.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-rulesdir": "^0.2.2",
     "eslint-plugin-sort-imports-requires": "^1.0.2",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
     "eslint-plugin-sql-template": "^2.0.0"
   },
   "devDependencies": {
-    "@uphold/github-changelog-generator": "^3.0.0",
-    "eslint": "~8.24.0",
-    "mocha": "^10.0.0",
+    "@uphold/github-changelog-generator": "^3.3.1",
+    "eslint": "^8.38.0",
+    "mocha": "^10.2.0",
     "pre-commit": "^1.2.2",
-    "prettier": "^2.7.1",
+    "prettier": "^2.8.7",
     "should": "^13.2.3"
   },
   "peerDependencies": {
-    "eslint": "~8.20.0",
-    "prettier": "^2.7.1"
+    "eslint": "^8.20.0",
```

Comment:
Note that we locked the minor version because eslint states that minor bumps can change the way eslint works, and alter the linting result of a project.